src/gameplayv2/indexCards.js

```diff
@@ -0,0 +1,29 @@
+// This file helps us load in the roles from the folder
```

Comment:
Let's not use this file. This was my hacky way to dynamically load in the files without having to register them in an index.js file, but this really isn't good to do!

Reply:
Feel free to ask me in a meeting sometime why this isn't great and the issues I ran into with this.